Gruntfile.js

```diff
@@ -3,12 +3,13 @@
 const fs = require('fs-extra');
 const path = require('path');
 const debug = require('debug');
+const grunt = require('grunt');
 
 const webpackConfig = require('./build/webpack.config');
 const webpackConfigTest = require('./test/test.webpack.config');
 const pkg = require('./package.json');
 
-const skipCodeSigning = process.argv.some(arg => arg.startsWith('--no-sign'));
+const skipCodeSigning = grunt.option('no-sign');
```

Comment:
Not like this. Remove it from this block and put it inside `module.exports = function(grunt)`, where `grunt` instance is available. You can move reading the config there as well.

Reply:
My bad - I should have noticed that 🤦‍♂